packages/components/src/components/button-toggle/button-toggle.scss

```diff
@@ -58,7 +58,7 @@
             &__button--active {
                 background-color: rgba(0, 0, 0, 0) !important;
                 z-index: 1;
-                padding: 0.8rem 0.6rem;
+                padding: 0.8rem 1.2rem;
```

Comment:
Let's not change the padding in the `components` package since this component is possibly being used elsewhere. Let's use `className` and give `padding` in `p2p` package.